apps/epl/src/epl.erl

```diff
@@ -48,7 +48,9 @@ trace_pid(Pid) ->
 
 proplist_to_json(Term) ->
     Obj = encode(Term, ej:new()),
-    ej:encode(Obj).
+    Data = [{<<"topic">>, <<"system-init">>},
```

Comment:
Please add Topic as an argument of epl:proplist_to_json so that different plugins can use this helper function with their custom topics.